katportalclient/client.py

```diff
@@ -296,7 +296,7 @@ def _get_sitemap(self, url):
                     result.update(response['client'])
                 except tornado.httpclient.HTTPError:
                     self._logger.exception("Failed to get sitemap!")
-                except json.JSONError:
+                except ValueError:
```

Comment:
@tockards See: https://github.com/not-kennethreitz/omnijson/blob/develop/omnijson/core.py#L69